docs/install.md

```diff
@@ -123,7 +123,7 @@ configured to talk to a remote Kubernetes cluster.
 
 Most cloud providers enable a feature called Role-Based Access Control - RBAC for short. If your cloud provider enables this feature, you will need to create a service account for Tiller with the right roles and permissions to access resources.
 
-Check the [Kubernetes Distribution Guide](kubernetes_distros.md) to see if there's any further points of interest on using Helm with your cloud provider. Also check out the guide on [Tiller and Role-Based Access Control](rbac.md) for more information on how to run Tiller in an RBAC-enabled Kubernetes cluster.
+Check the [Kubernetes Distribution Guide](kubernetes-distribution-guide) to see if there's any further points of interest on using Helm with your cloud provider. Also check out the guide on [Tiller and Role-Based Access Control](rbac.md) for more information on how to run Tiller in an RBAC-enabled Kubernetes cluster.
```

Comment:
would you mind adding a hashtag to the beginning of the internal link? If I re-build the site with this PR, the link 404s.

This is one of the main reasons why we migrated the docs over to helm-www for Helm 3. The redirection tasks are... finnicky... to set up and test.

Reply:
Ugh, sorry. Was eyeballing how another link looked to be listed. I think I've fixed it. (And yes, I'll admit I didn't test the gulp tasks locally.)